src/short-url/short-url.service.spec.ts

```
@@ -1,12 +1,21 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { ShortUrlService } from './short-url.service';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { ShortUrl } from './entities/short-url.entity';
+import { ShortUrlController } from './short-url.controller';
```

ESLint

[error] 'ShortUrlController' is defined but never used.

(@typescript-eslint/no-unused-vars)